crates/weechat/build.rs

```diff
@@ -0,0 +1,37 @@
+/// Keep only versions that break signatures in crate
+#[allow(unused)]
+#[repr(u64)]
+enum WeechatApiVersions {
+    V4_1_0 = 20230908,
+    V4_2_0 = 20240105,
+}
+
+fn main() {
+    println!("cargo::rerun-if-changed=build.rs");
+    println!("cargo::rerun-if-env-changed=WEECHAT_BUNDLED");
+    println!("cargo::rerun-if-env-changed=WEECHAT_PLUGIN_FILE");
+    println!("cargo::rustc-check-cfg=cfg(weechat410)");
+    println!("cargo::rustc-check-cfg=cfg(weechat420)");
+
+    let (version, _) = std::str::from_utf8(weechat_sys::WEECHAT_PLUGIN_API_VERSION)
+        .unwrap()
```

Comment:
Can we replace those unwraps with `expects()`? It's much nicer to see a sensible error message, even if we panic.